Moderation-Policy.md

```diff
@@ -241,8 +241,8 @@ removed from the Node.js GitHub organization.
 
 The Node.js Moderation Team is tasked with enforcement of this policy.
 
-Moderation team members are Collaborators nominated by either the TSC or
-CommComm and must be approved by *both* committees with annual recertification.
+Moderation team members are Collaborators who self-nominate or are nominated by either the TSC or
+CommComm, and must be approved by *both* committees with annual recertification.
```

Comment:
Micro-nit, totally non-blocking: This could benefit from shorter sentences. Maybe this?:

> Moderation team members are Collaborators who self-nominate or are nominated by either the TSC or
> CommComm. Team members must be approved by *both* committees with annual recertification.

Reply:
Oh, another non-blocking suggestion: Maintain the wrapping of text at 80 chars.